app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  validates :text, presence: true
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end

```

Comment:
[OPTIONAL] After using the rails active callbacks after_save method, kindly consider making the update_comments_counter method private to make your code simple and up to standard.